src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -15,51 +16,91 @@ void EnViewer_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnViewer_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnViewer_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_01000000;
-extern UNK_TYPE D_0404D4E0;
-extern UNK_TYPE D_06000450;
-extern UNK_TYPE D_060005B4;
-extern UNK_TYPE D_060008A0;
-extern UNK_TYPE D_06000BC8;
-extern UNK_TYPE D_06000F54;
-extern UNK_TYPE D_06001410;
-extern UNK_TYPE D_060014F4;
-extern UNK_TYPE D_06001D28;
-extern UNK_TYPE D_06002574;
-extern UNK_TYPE D_06002650;
-extern UNK_TYPE D_06002928;
-extern UNK_TYPE D_060029CC;
+// EnView_AnimFunc x3
+void func_80B2A300(EnViewer* this, GlobalContext* globalCtx, FlexSkeletonHeader* skeletonHeaderSeg,
+                   AnimationHeader* animationSeg);
+void func_80B2A448(EnViewer* this, GlobalContext* globalCtx, FlexSkeletonHeader* skeletonHeaderSeg,
+                   AnimationHeader* animationSeg);
```

Comment:
you probably dont need most of these declarations if I had to guess

Reply:
The AnimFuncs seeem to need a declaration, but yeah many of the other functions did not